src/ui/sass/modules/_SearchBar.scss

```diff
@@ -216,6 +216,11 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
     transform-origin: 50% 50%;
     animation: dash 1.4s ease-in-out infinite;
   }
+
```

Comment:
The problem occurs when the icon transitions while the loading indicator animation is running. Since the loading indicator was always running in the background, this was always happening. By setting the animated icon to `display: none` while it is inactive, it is no longer possible for the loading indicator to be running while the icon is transitioning from the magnifying glass to the yext logo